packages/fiber/src/core/renderer.ts

```diff
@@ -243,14 +243,6 @@ function createRenderer<TCanvas>(roots: Map<TCanvas, Root>) {
       invalidateInstance(instance)
     })
 
-    if (rootState.internal && instance.raycast && prevHandlers !== localState.eventCount) {
-      // Pre-emptively remove the instance from the interaction manager
-      const index = rootState.internal.interaction.indexOf(instance as unknown as THREE.Object3D)
-      if (index > -1) rootState.internal.interaction.splice(index, 1)
-      // Add the instance to the interaction manager only when it has handlers
-      if (localState.eventCount) rootState.internal.interaction.push(instance as unknown as THREE.Object3D)
-    }
```

Comment:
Does this break reactivity with handler props? Maybe we should add this back but have it run when the instance is mounted or parented.